Gemfile

```diff
@@ -1,4 +1,4 @@
-raise "Ruby versions less than 2.2.2 are unsupported!" if RUBY_VERSION < "2.2.2"
+raise "Ruby versions less than 2.3.5 are unsupported!" if RUBY_VERSION < "2.3.5"
```

Comment:
@jrafanie Is this...ok? Sanity check.

Reply:
No, I think this needs to be < 2.3.1 since the oldest 2.3 will be on downstream appliances with the 2.3.1 package cc @simaishi (for confirmation)

Reply:
Right, `2.3.1` needs to stay as the supported version until downstream appliances get newer ruby.

Reply:
`<= 2.3.1` (don't forget the `=` sign)